.env.example

```diff
@@ -1,2 +1,4 @@
 BOT_TOKEN='YOUR_BOT_TOKEN_HERE'
 GUILD_ID='YOUR_GUILD_ID_HERE'
+IMAGE_API='YOUR_OCR_SPACE_API_KEY_HERE'
+IMAGE_URL='https://api.ocr.space/parse/image'
```

Comment:
We should try and look at a solution that doesn't use an external API to not have to rely on one.

Reply:
Well currently the best api, its free for 25k and is a great recognition, i looked at some of api's and npm packages before but this is the best.